tests/snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py

```diff
@@ -361,6 +361,7 @@ def test_pfcwd_basic_multi_lossless_prio_restart_service(snappi_api,
 
         logger.info('Port dictionary:{}'.format(ports_dict))
         for duthost in list(set([snappi_ports[0]['duthost'], snappi_ports[1]['duthost']])):
+            up_bgp_neighbors = duthost.get_bgp_neighbors_per_asic("established")
```

Comment:
@rraghav-cisco  - the BGP state check is ONLY if the duthost is multi.asic. Is that not true for other systems?

Reply:
I am focusing only on T2/multiasic at the moment. The non-mAsic systems are in the else block. I will have to check that when t0/t1/t2 are all combined.

Reply:
Also, the internal BGP routes are not present for T0/T1. Only T2 has them. The other checks - services and interfaces will be needed for T0/T1 as well.

Reply:
as discussed, @rraghav-cisco pls open a separate PR to handle T0/T1 case.